modules/ossm-about-concepts-kiali.adoc

```diff
@@ -0,0 +1,69 @@
+
+In addition, Kiali depends on external services and components provided by the container application platform and Istio.
+
+Red Hat Service Mesh (Istio):: Istio is a Kiali requirement. Istio is the component that provides and controls the service mesh. Although Kiali and Istio can be installed separately, Kiali depends on Istio and will not work if it is not present. Kiali needs to retrieve Istio data and configurations, which are exposed through Prometheus and the {product-title} cluster API.
+
+Prometheus:: A dedicated Prometheus instance optional. When Istio telemetry is enabled, metrics data are stored in Prometheus. Kiali uses this Prometheus data to determine the mesh topology, display metrics, calculate health, show possible problems, and so on. Kiali communicates directly with Prometheus and assumes the data schema used by Istio Telemetry. Prometheus is an Istio dependency and a hard dependency for Kiali, and many of Kiali's features will not work without Prometheus.
+
+Cluster API:: Kiali uses the API of the {product-title} (cluster API) to fetch and resolve service mesh configurations. Kiali queries the cluster API to retrieve, for example, definitions for namespaces, services, deployments, pods, and other entities. Kiali also makes queries to resolve relationships between the different cluster entities. The cluster API is also queried to retrieve Istio configurations like virtual services, destination rules, route rules, gateways, quotas, and so on.
+
+Tracing:: Tracing is optional, but when you install {DTProductName} and Kiali is configured, the Kiali console includes a tab to display distributed tracing data, and tracing integration on the graph itself. Note that tracing data will not be available if you disable Istio’s distributed tracing feature. Also note that the user must have access to the namespace where the Istio control plane is installed to view tracing data.
+
+Grafana:: Grafana is optional. When available, the metrics pages of Kiali display links to direct the user to the same metric in Grafana. Note that user must have access to the namespace where the Istio control plane is installed to view links to the Grafana dashboard and view Grafana data.
+
+== Kiali features
+
+The Kiali console is integrated with Red Hat Service Mesh and provides the following capabilities:
+
+Health:: Quickly identify issues with applications, services, or workloads.
+
+Topology:: Visualize how your applications, services, or workloads communicate via the Kiali graph.
+
+Metrics::  Predefined metrics dashboards let you chart service mesh and application performance for Go, Node.js. Quarkus, Spring Boot, Thorntail and Vert.x. You can also create your own custom dashboards.
+
+Tracing:: Integration with Jaeger lets you follow the path of a request through various microservices that make up an application.
```

Comment:
Should we change Jaeger to Tempo as that's what we support?

Reply:
@gwynnemonahan yes, I think we can say Tempo here.

Reply:
Thank you for confirming!